collections/infrastructure/roles/clustershell/README.md

```diff
@@ -20,14 +20,101 @@ Once role has been deployed, check groups using:
 nodeset -LL
 ```
 
+## Custom nodeset Map
+
+By default, the role generates ClusterShell groups based on the Ansible inventory.
+
+However, you can generate custom groups with other names and also clusters (see man 5 groups.conf) using the role.
+
+If you want to customize, you must configure as below:
+
+1. Change the ``clustershell_custom_map`` variable to true inside the ``defaults`` folder.
```

Comment:
Okay, I see my mistake here. 

I'm going to improve this question so that you don't even use the variable, as you commented below. 
Just check if the inventory exists. 

I'll also rename the inventory to something that reflects the role.